src/ContainerAppHelper.ts

```diff
@@ -293,7 +293,7 @@ export class ContainerAppHelper {
         try {
             let command = `az containerapp env list -g ${resourceGroup} --query "[0].name"`
             let executionResult = await util.execute(command);
-            return executionResult.exitCode === 0 ? executionResult.stdout : null;
+            return executionResult.exitCode === 0 ? executionResult.stdout.replace(/(\r\n|\n|\r)/gm, "") : null;
```

Comment:
Nit: should we consider making this at the util.execute level? It feels like this could be a problem in all cases potentially?

Reply:
+1. We can probably add this `replace` method to stdout [here](https://github.com/Azure/container-apps-deploy-action/blob/main/src/GithubActionsToolHelper.ts#L52) so that way we know the standard output does not have `/r` or `/n` as such

Reply:
Makes sense! Moving it in next commit